x-pack/plugins/cases/docs/openapi/components/schemas/create_case_request.yaml

```
@@ -24,6 +24,7 @@ properties:
   description:
     description: The description for the case.
     type: string
+    maximum: 30000
```

If the intention is to indicate that there can be a maximum of 30000 characters in the string, I think we want maxLength per https://swagger.io/docs/specification/data-models/data-types/#string. "maximum" is typically used for numeric data types.
```suggestion
    maxLength: 30000
```